src/OrchardCore.Modules/OrchardCore.Contents/Services/DefaultContentsAdminListFilter.cs

```diff
@@ -89,6 +89,11 @@ public async Task FilterAsync(ContentOptionsViewModel model, IQuery<ContentItem>
                 query.With<ContentItemIndex>(x => x.ContentType.IsIn(listableTypes.Select(t => t.Name).ToArray()));
             }
 
+            if(!await _authorizationService.AuthorizeAsync(_httpContextAccessor.HttpContext.User, Permissions.ViewAllAdminContentList))
+            {
+                query.With<ContentItemIndex>(x => x.Owner == _httpContextAccessor.HttpContext.User.Identity.Name);
```

Comment:
I like this solution to listing only owned items, but I'm not sure about the name / description of it.

Reply:
Find one 🙈 😛

Reply:
public static readonly Permission ViewRestrictedItemsAdminContentList = new Permission("ViewAllAdminContentList", "View restricted/unrestricted content item(s) in admin content list");